perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -867,6 +867,26 @@ sub update_send_failed {
     } );
 }
 
+=head2 updates_sent_to_body
+
+Returns 1 if updates left on this report will be sent to any of the receiving
+bodies by some mechanism. Right now that mechanism is Open311.
+
+=cut
+
+sub updates_sent_to_body {
+    my $self = shift;
+    return unless $self->send_method_used && $self->send_method_used eq 'Open311';
+
+    # Some bodies only send updates *to* FMS, they don't receive updates.
+    # NB See also the list in bin/send-comments
+    my $excluded = qr{Lewisham|Oxfordshire};
+
+    my @bodies = values %{ $self->bodies };
+    my @updates_sent = grep { $_ } map { $_->send_comments && !($_->name =~ /$excluded/) } @bodies;
```

Comment:
`s'grep { $_ } map'grep'` :)

Reply:
Does this need to check $_->send_method is still Open311 as well? If the body had potentially switched back to email, I mean. OTOH, if they're temporarily on Noop due to brokenness their end...

Reply:
Good idea, I've updated it to hide the message if the send method is `Open311` or `Noop`.